torch/testing/_internal/common_distributed.py

```diff
@@ -590,6 +590,9 @@ def destroy_pg_upon_exit(self) -> bool:
     def world_size(self) -> int:
         return DEFAULT_WORLD_SIZE
 
+    @world_size.setter
+    def world_size(self, value):
+        self._world_size = value
```

Comment:
new line

Also, how would this setter interact with the world_size function above?